src/charm.py

```diff
@@ -388,6 +420,7 @@ def _on_oathkeeper_pebble_ready(self, event: PebbleReadyEvent) -> None:
         """Event Handler for pebble ready event."""
         self._patch_statefulset()
         self._handle_status_update_config(event)
+        self._update_oathkeeper_info_relation_data(event)
```

Comment:
Question: Should this call be moved somewhere else? The reason for this is that if `_on_ingress_revoked` is called, the configuration will not be updated (because the info will still be in the databag). e.g. should we try to update on `update_status` so that we know that we will be eventually consistent?